src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/codeSmell/UsageOfSilenceOperatorInspector.java

```diff
@@ -33,8 +34,8 @@
  */
 
 public class UsageOfSilenceOperatorInspector extends BasePhpInspection {
-    // configuration flags automatically saved by IDE
-    public boolean RESPECT_CONTEXT = true;
+    // Inspection options.
+    public boolean optionRespectContext = false;
```

Comment:
Yes, I do agree that is better implements it first (with original variable names), then care about variable changes in another context (it can be done in another moment, another PR). It because I need see if it is possible currently too (for now I can think about a workaround only, and I like to avoid that).